traitsui/wx/led_editor.py

```diff
@@ -41,16 +32,19 @@
 }
 
 
-class _LEDEditor(Editor):
+class LEDEditor(Editor):
     """ Traits UI 'display only' LED numeric editor.
     """
 
+    #: The alignment of the numeric text within the control.
+    alignment = Enum("right", "center", "left")
+
     def init(self, parent):
         """ Finishes initializing the editor by creating the underlying toolkit
             widget.
         """
         self.control = LEDNumberCtrl(parent, -1)
-        self.control.SetAlignment(LEDStyles[self.factory.alignment])
+        self.control.SetAlignment(LEDStyles[self.alignment])
```

Comment:
Same here, I think we can defer making `alignment` mutable in a separate PR / issue and restore the original code which uses the factory value.

(I bet one probably needs to `sync_value` method like this https://github.com/enthought/traitsui/blob/7841ca7bf2742f7087f1fc9150ace89c57db63dd/traitsui/qt4/list_str_editor.py#L114 so that the changes are handled on the event loop and so that the value on the editor factory is actually transferred to the editor.)

Reply:
> (I bet one probably needs to sync_value method like this...

Please ignore this half of the comment. The `sync_value` is used for synchronizing a trait on the editor with a trait on the object being edited. But the first half of the comment about defer making `alignment` mutable still stands.